crates/robonode-client/src/enroll.rs

```diff
@@ -118,11 +157,17 @@ mod tests {
             public_key: b"123",
             liveness_data_signature: b"signature",
         };
+        let sample_response = serde_json::json!({
+            "scanResultBlob": "scanResultBlob"
+        });
+
+        let expected_response: EnrollResponse =
```

Comment:
This change in the tests actually masks that the new code fails to work with the old robonode; instead, add a new test

Reply:
Fixed - https://github.com/humanode-network/humanode/pull/1026/commits/77f2c1057bef108e7113b68d244e68307528ba9c

Reply:
https://github.com/humanode-network/humanode/pull/1026/commits/a240fb477601465dd36d3823ae7afac37d301616